### ISSUE TITLE
add download function

### ISSUE THREAD

**Issue by matthiasdusch** [OGGM/rgitools#30]
Should we add the OGGM download function (e.g. what @fmaussion is working on here https://github.com/OGGM/oggm/pull/923) to RGITOOLS, so people don't have to install OGGM?

**Comment by fmaussion** [comment]
Yes - removing dependency to OGGM would be good one day - but I'd say it's less urgent than other things..
